src/hb-ot-color-colr-table.hh

```diff
@@ -895,6 +900,14 @@ struct ClipBoxFormat1
     return_trace (c->check_struct (this));
   }
 
+  void get_clip_box (ClipBoxData &clip_box, const VarStoreInstancer &instancer) const
```

Comment:
This doesn't use the instancer.

Reply:
If you need it for API signature, mark it HB_UNUSED.

Reply:
Yes, thats the non-variable implementation of the api. I'll add the unused